docs/docs/recipes.md

```diff
@@ -265,6 +265,46 @@ export default ({ pageContext: { dog } }) => (
 
 There are so many ways to add styles to your website; Gatsby supports almost every possible option, through official and community plugins.
 
+### Using global CSS files without a Layout component
+
+#### Prerequisites
+
+- An existing [Gatsby site](/docs/quick-start/) with an index page component
+- A `gatsby-browser.js` page 
+
+#### Directions
+
+1. Create a global CSS file as `src/styles/global.css` and paste the following into the file:
+
+```css:title=src/styles/styles/global.css
+html {
+  background-color: lavenderblush;
+}
+
+p {
+  color: peachpuff;
+}
+```
+
+2. Import the global CSS file in the `gatsby-config.js` file such as the following:
```

Comment:
```suggestion
2. Import the global CSS file in the `gatsby-browser.js` file such as the following:
```

Reply:
Oh yes, sorry about that! Thanks for pointing this out. Applying the suggestions right away!